doc/source/examples/dem/small-scale-rotating-drum/small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,182 @@
+
+This example of Lethe-DEM is to simulate the behaviour of dry granular flow in a small scale rotating drum. The discrete element method DEM will be responsible of describing the behaviour of particles. DEM parameters are described in Lethe-DEM documentation, i.e. `DEM parameters <../../../parameters/dem/dem.html>`_ for more detailed information.
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Three-dimensional problem
+- Rotational boundary
+- Load-balancing
+
+
+Location of the examples
+------------------------
+ ``/examples/dem/small_scale_rotating_drum/small_rotating_drum_dem.prm``
+
+
+Description of the case
+-----------------------
+
+This example simulates a cascading regime in a small scale rotating drum. First, we first use Lethe-DEM to fill the bed with 20000 particles. We enable check-pointing in order to write the DEM checkpoint files which will be used as the starting point of the DEM simulations . Then, the solver ``dem_3d`` is used to simulate the behaviour of dry granular flow within the rotating drum.
+
+
+Parameter file
+--------------
+
+Mesh
+~~~~~
+
+In this example, we choose a ``cylinder`` grid type to create a cylinder. Grid arguments are the radius of the cylinder (0.056 m) and half-length (0.051m) , respectively.  The grid is refined 3 times. The ``expand particle-wall contact search`` is used in concave geometries to enable extended particle-wall contact search with boundary faces of neighbor cells for particles located in each boundary cell (for more details  see `Rotating Drum example <../rotating-drum/rotating-drum.html>`_).
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 				= dealii
+        set grid type      	     				= cylinder
+        set grid arguments       				= 0.056:0.051
+        set initial refinement   				= 3
+    end
+
+
+Packing information
+~~~~~~~~~~~~~~~~~~~~
+
+An insertion box is defined inside the cylindrical domain, inserting 8000 particles every 0.5 seconds while the cylinder is at rest.
+
+.. code-block:: text
+
+
+    subsection insertion info
+    	set insertion method				        = non_uniform
+    	set inserted number of particles at each time step      = 8000
+    	set insertion frequency            		 	= 100000
+    	set insertion box minimum x            	 	= -0.05
+    	set insertion box minimum y            	        = 0.
+    	set insertion box minimum z            	        = -0.04
+    	set insertion box maximum x            	        = 0.05
+    	set insertion box maximum y           	 	= 0.04
+    	set insertion box maximum z            	        = 0.04
+    	set insertion distance threshold			= 1.1
+    	set insertion random number range			= 0.05
+    	set insertion random number seed			= 19
+    end
+
+Restart files are written once the packing ends.
+
+Lagrangian physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles are monodispersed with a radius 0.0015 m  and a density of 2500 kg/m3, respectively.
```

Comment:
I would explain how to do it. There are many parameters in this section with no explanation provided.